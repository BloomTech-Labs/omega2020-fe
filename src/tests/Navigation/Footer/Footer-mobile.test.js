import React from 'react';
import renderer from 'react-test-renderer';
import { configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import '@testing-library/jest-dom/extend-expect';
import { Typography, Box, CardHeader } from '@material-ui/core';
import FooterMobile from '../../../components/Navigation/Footer/Footer-mobile';

configure({ adapter: new Adapter() });


describe('the length of the material ui and match the snapshot of the foot mobile component', () => {

    const wrapper = mount(<FooterMobile />)

    it('snapshot the component', () => {
        const Footer = renderer.create(<FooterMobile />).toJSON();
        expect(Footer).toMatchSnapshot();

    })

    it('render the length of the material ui', () => {

        expect(wrapper.find(Typography).length).toEqual(0)
        expect(wrapper.find(Box).length).toEqual(5)
        expect(wrapper.find(CardHeader).length).toEqual(3)

    })

})