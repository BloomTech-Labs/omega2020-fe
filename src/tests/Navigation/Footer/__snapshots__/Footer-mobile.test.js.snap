// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the length of the material ui and match the snapshot of the foot mobile component snapshot the component 1`] = `
<div
  className="MuiBox-root MuiBox-root-11 makeStyles-root-1"
>
  <dark
    className="makeStyles-root-1"
  >
    <div
      className="MuiBox-root MuiBox-root-12 makeStyles-leftContainer-2"
    >
      <div
        className="MuiCardHeader-root"
      >
        <div
          className="MuiCardHeader-content"
        />
        <div
          className="MuiCardHeader-action"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            style={
              Object {
                "fontSize": 30,
                "paddingTop": 10,
              }
            }
            viewBox="0 0 24 24"
          >
            <path
              d="M17 3H7c-1.1 0-1.99.9-1.99 2L5 21l7-3 7 3V5c0-1.1-.9-2-2-2z"
            />
          </svg>
        </div>
      </div>
    </div>
    <div
      className="MuiBox-root MuiBox-root-13 makeStyles-centerContainer-4"
    >
      <div
        className="MuiCardHeader-root"
      >
        <div
          className="MuiCardHeader-avatar"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            style={
              Object {
                "fontSize": 30,
                "paddingTop": 10,
              }
            }
            viewBox="0 0 24 24"
          >
            <path
              d="M4 8h4V4H4v4zm6 12h4v-4h-4v4zm-6 0h4v-4H4v4zm0-6h4v-4H4v4zm6 0h4v-4h-4v4zm6-10v4h4V4h-4zm-6 4h4V4h-4v4zm6 6h4v-4h-4v4zm0 6h4v-4h-4v4z"
            />
          </svg>
        </div>
        <div
          className="MuiCardHeader-content"
        />
      </div>
    </div>
    <div
      className="MuiBox-root MuiBox-root-14 makeStyles-rightContainer-3"
    >
      <div
        className="MuiBox-root MuiBox-root-15 makeStyles-icons-5"
      >
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-avatar"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              style={
                Object {
                  "fontSize": 30,
                  "paddingTop": 10,
                }
              }
              viewBox="0 0 24 24"
            >
              <path
                d="M10 20h4V4h-4v16zm-6 0h4v-8H4v8zM16 9v11h4V9h-4z"
              />
            </svg>
          </div>
          <div
            className="MuiCardHeader-content"
          />
        </div>
      </div>
    </div>
  </dark>
</div>
`;
